app/src/androidTest/java/de/codecentric/androidtestktx/espresso/extensions/Matchers.kt

```diff
@@ -69,32 +68,10 @@ fun firstVisibleItem(matcher: Matcher<View>): Matcher<View> {
 /**
  * Tests if the argument is a string that contains a substring.
  */
-class StringContainsLowerCase(substring: String) : SubstringMatcher(substring) {
+class StringContainsLowerCase(substring: String) : SubstringMatcher("containing lower case", true, substring) {
```

Comment:
Hello @pwargulak @jdolkowski @druchniewicz , is this ticket still valid to be reviewed.